include/alpaka/acc/AccGpuUniformCudaHipRt.hpp

```diff
@@ -224,8 +228,22 @@ namespace alpaka
         {
             ALPAKA_FN_HOST static auto getAccName() -> std::string
             {
-                return std::string("AccGpu") + TApi::name + "Rt<" + std::to_string(TDim::value) + ","
-                       + core::demangled<TIdx> + ">";
+#    if BOOST_COMP_CLANG
+#        pragma clang diagnostic push
+#        pragma clang diagnostic ignored "-Wexit-time-destructors"
+#    endif
+                static std::string const accName =
+#    ifdef __cpp_lib_format
+                    std::format("AccGpu{}Rt<{},{}>", TApi::name, TDim::value, core::demangled<TIdx>);
+#    else
+                    (std::ostringstream()
```

Comment:
To workaround the nvcc error here 

```
error: class "std::basic_ostream<char, std::char_traits<char>>" has no member "str"
                          .str();
                           ^
```

I would try if `std::string(<ostringstream>)` works